### DISCUSSION TITLE
國際書展

### DISCUSSION THREAD

**Discussion by nelson910402** [1101-GS4719-C/Python-Community-GS4719-C#1929] [General]
大家有沒有去前幾週的台北國際書展，當初進去是兒童展區的出口，還以為進錯展了。書展內有很多不同種的書，連外國的原文書也有。比較不滿意的點是，明明有好多大學都有設攤位推薦書籍，為什麼中央沒有參與！
另外分享一下，我掏到的好書![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/160851751/b47d6a17-3437-4da8-a19c-35969282c8b7)JoJo的美術介紹，非常期待裡面的內容

**Comment by woohsin** [comment]
我只知道有個出版社不能用文化幣，所以我花了好多錢...
